WORKSPACE

```diff
@@ -122,6 +122,76 @@ go_repository(
     version = "v0.0.0-20210628180205-a41e5a781914",
 )
 
+go_repository(
```

Comment:
These will need to be in the go_repositories macro.

Reply:
I think a full fledge `deps.bzl` is needed for these `go_repository` targets, and optionally opt-in for those who are interested in using staticcheck? wdyt?